specifyweb/frontend/js_src/lib/components/Formatters/Formatter.tsx

```diff
@@ -145,51 +147,142 @@ function Definitions({
     }
   }, [needFormatter, handleChange, formatter.definition.fields]);
 
+  const [showConditionalField, setShowConditionalField] = React.useState(
+    trimmedFields.map(() => false)
+  );
+
   return table === undefined ? null : (
     <div className="flex flex-col gap-4 divide-y divide-gray-500 [&>*]:pt-4">
       {trimmedFields.map(({ value, fields }, index) => (
-        <div className="flex flex-col gap-2" key={index}>
+        <div
+          className={`flex ${
+            showConditionalField[index] || !hasCondition
+              ? 'flex-col'
+              : 'items-center'
+          } gap-2`}
+          key={index}
+        >
           {hasCondition && (
             <Label.Block>
-              {resourcesText.conditionFieldValue()}
-              <Input.Text
-                isReadOnly={isReadOnly}
-                value={value ?? ''}
-                onValueChange={(value): void =>
-                  handleChanged(
-                    {
-                      value: value.length === 0 ? undefined : value,
-                      fields,
-                    },
-                    index
-                  )
-                }
-              />
-              <span>
-                {index === 0
-                  ? resourcesText.elseConditionDescription()
-                  : resourcesText.conditionDescription()}
-              </span>
+              {showConditionalField[index]
+                ? resourcesText.conditionFieldValue()
+                : null}
+              <div className="flex items-center gap-2">
```

Comment:
is `items-center` needed here??

Reply:
yes 
![Screenshot 2024-01-11 at 10 03 21 AM](https://github.com/specify/specify7/assets/108160931/a7c49ac4-63d9-4537-86cf-3b698603f8ca)
VS
![Screenshot 2024-01-11 at 10 03 30 AM](https://github.com/specify/specify7/assets/108160931/c114a243-e27c-49e9-9793-9c6995d9779f)